docs/docs/cmd/planner/task/task-add.md

```diff
@@ -112,3 +112,121 @@ Adds a Microsoft Planner task with the name _My Planner Task_ for plan with the
 ```sh
 m365 planner task add --title "My Planner Task" --planId "8QZEH7b3wkSbGQobscsM5gADCBa" --bucketId "IK8tuFTwQEa5vTonM7ZMRZgAKdna" --appliedCategories "category1,category3" --previewType "noPreview"
 ```
+
+## Response
+
+=== "JSON"
+
+    ```json
+    {
+      "planId": "oUHpnKBFekqfGE_PS6GGUZcAFY7b",
+      "bucketId": "vncYUXCRBke28qMLB-d4xJcACtNz",
+      "title": "Important task",
+      "orderHint": "8585269241124027581",
+      "assigneePriority": "",
+      "percentComplete": 50,
+      "startDateTime": "2023-01-20T00:00:00Z",
+      "createdDateTime": "2023-01-25T21:39:33.0748226Z",
+      "dueDateTime": "2023-02-15T00:00:00Z",
+      "hasDescription": true,
+      "previewType": "automatic",
+      "completedDateTime": null,
+      "completedBy": null,
+      "referenceCount": 0,
+      "checklistItemCount": 0,
+      "activeChecklistItemCount": 0,
+      "conversationThreadId": null,
+      "priority": 5,
+      "id": "D-ys8Ef4kEuwYG4r68Um3pcAAe9M",
+      "createdBy": {
+        "user": {
+          "displayName": null,
+          "id": "b2091e18-7882-4efe-b7d1-90703f5a5c65"
+        },
+        "application": {
+          "displayName": null,
+          "id": "31359c7f-bd7e-475c-86db-fdb8c937548e"
+        }
+      },
+      "appliedCategories": {},
+      "assignments": {},
+      "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+      "references": {},
+      "checklist": {}
+    }
```

Comment:
This response is only achieved when they use the options `description` or `previewType`. Let's include both outputs similarly like `planner plan add`

[The docs article](https://pnp.github.io/cli-microsoft365/cmd/planner/plan/plan-add/)
[The MarkDown file location](https://github.com/pnp/cli-microsoft365/blob/main/docs/docs/cmd/planner/plan/plan-add.md?plain=1)